RELEASENOTES.md

```diff
@@ -1,18 +1,40 @@
 # Release Notes
 
-![Ripple](docs/images/ripple.png)
+![XRP](docs/images/xrp-text-mark-black-small@2x.png)
 
-This document contains the release notes for `rippled`, the reference server implementation of the Ripple protocol. To learn more about how to build and run a `rippled` server, visit https://ripple.com/build/rippled-setup/
+This document contains the release notes for `rippled`, the reference server implementation of the Ripple protocol. To learn more about how to build, run or update a `rippled` server, visit https://xrpl.org/install-rippled.html
 
-> **Do you work at a digital asset exchange or wallet provider?** 
-> 
-> Please [contact us](mailto:support@ripple.com). We can help guide your integration.
+ 
+Have new ideas? Need help with setting up your node? Come visit us [here](https://github.com/ripple/rippled/issues/new/choose)
 
-## Updating `rippled`
+# Releases
 
-If you are using Red Hat Enterprise Linux 7 or CentOS 7, you can [update using `yum`](https://ripple.com/build/rippled-setup/#updating-rippled). For other platforms, please [compile from source](https://wiki.ripple.com/Rippled_build_instructions).
+## Version 1.5.0
+
+The `rippled` 1.5.0 release introduces several improvements and new features, including support for gRPC API, API versioning, UNL propagation via the peer network, new RPC methods `validator_info` and `manifest`, augmented `submit` method, improved `tx` method, improved CLI parsing, improved protocol-level handshaking protocol, improved package building and various other minor bug fixes and improvements.
+
+This release also introduces two new amendments: `fixQualityUpperBound` and `RequireFullyCanonicalSig`.
+ 
+**New and Updated Features**
+- Native [gRPC API](https://grpc.io/) API support. Currently, this API provides a subset of the full `rippled` [API](https://xrpl.org/rippled-api.html). You can enable the gRPC API on your server with a new configuration stanza. 
+- API Versioning which allows for future breaking change of RPC methods to co-exists with existing versions.
+- Nodes now receive and broadcast UNLs over the peer network under various conditions. see `rippled` [#3072](https://github.com/ripple/rippled/pull/3072)
+- Augmented `submit` method to include additional details on the status of the command.
+- Improved `tx` method to allow the search to specify a range of ledgers using the `min_ledger` and `max_ledger` fields in the request.
+- new `validator_info` method which returns information pertaining to the current validators keys, manifest sequence, and domain.
+- new `manifest` method which looks up manifest information for the specified key (either master or ephemeral).
+- Introduce handshake protocol for compression negotiation (compression is not implemented at this point) and other minor improvements. 
+- Remove various old conditionals introduced by amendments.
+- Add `getRippledInfo` to packages to include info gathering script in `rippled` Linux packages.
```

Comment:
This sentence reads oddly to me.  Consider instead...

"Add `getRippledInfo` info gathering script to `rippled` Linux packages."

Reply:
Good catch! Updated